docfx_yaml/extension.py

```diff
@@ -1238,6 +1253,9 @@ def convert_module_to_package_if_needed(obj):
     if len(toc_yaml) == 0:
         raise RuntimeError("No documentation for this module.")
 
+    # Ensure names align with the last bits of the uid
+    check_name_with_uid(toc_yaml)
```

Comment:
We're mixing API types. `group_by_package` returns a value and `check_name_with_uid` modifies the value directly. We should pick one.

Reply:
Noted. I'll stick with modifying the value directly rather than returning the toc.

Reply:
Correction, I'll return the value and remove `check_name_with_uid`.